test/distributed/org/apache/cassandra/fuzz/topology/AccordTopologyMixupTest.java

```diff
@@ -240,13 +245,39 @@ public String createSchema()
 
     private static class AccordState extends State<Spec>
     {
+        private final Int2ObjectHashMap<String> instanceEpochState = new Int2ObjectHashMap<>();
         private final ListenerHolder listener;
 
         public AccordState(RandomSource rs)
         {
             super(rs, AccordTopologyMixupTest::createSchemaSpec, AccordTopologyMixupTest::cqlOperations);
 
             this.listener = new ListenerHolder(this);
+            this.preActions.add(this::populateEpochState);
```

Comment:
before every step we collect a snapshot of epoch state, if a step fails we show the status *before* the step